src/Stnvh/Partial/Zip.php

```diff
@@ -64,9 +64,6 @@ private function receiveData($ch, $data) {
 	 * @return void
 	 */
 	public function __construct($url, $file = false) {
-		set_time_limit(15);
-		ob_start();
-
```

Comment:
ob_start suppresses all output from my script
i think it must be done by selectively ignoring errors with "@" or just checking existence before using array index.

set_time_limit prevents to make a long-running import scripts, for example i have a 2GB zip with thousands images on remote server and i need to get only several hundreds of it. But it't time consuming.
I think it must been better way to check http range support - for example - compare request & response range headers.